cli/command/secret/utils.go

```diff
@@ -18,3 +18,30 @@ func getSecretsByName(ctx context.Context, client client.APIClient, names []stri
 		Filters: args,
 	})
 }
+
+func getCliRequestedSecretIDs(ctx context.Context, client client.APIClient, names []string) ([]string, error) {
+	ids := names
+
+	// attempt to lookup secret by name
+	secrets, err := getSecretsByName(ctx, client, ids)
+	if err != nil {
+		return nil, err
+	}
+
+	lookup := make(map[string]int)
+	for _, id := range ids {
+		lookup[id] = 1
```

Comment:
The standard seems to be `map[string]struct{}` instead of `map[string]int`. And then `lookup[id] = struct{}{}`.

I think it symbolizes that it has exactly zero meaning but "exists".